### PR TITLE
go_md2man: new recipe

### PR DESCRIPTION
**Pull request by sfanxiang**
This recipe is a dependency of #2078.

### CODE REVIEW

File: dev-go/go-md2man/go_md2man-1.0.7.recipe [hunk 1]
```diff
@@ -0,0 +1,60 @@
+SUMMARY="Uses blackfriday to process markdown into man pages"
+DESCRIPTION="go-md2man uses blackfriday to process markdown into man pages"
+HOMEPAGE="https://github.com/cpuguy83/go-md2man"
+COPYRIGHT="2017 Brian Goff"
```
Comment (alaviss) [inline dev-go/go-md2man/go_md2man-1.0.7.recipe]:
```diff
-COPYRIGHT="2017 Brian Goff"
+COPYRIGHT="2017 Brian Goff
+	2017 Russ Ross"

File: dev-go/go-md2man/go_md2man-1.0.7.recipe [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+LICENSE="MIT"
```
Comment (alaviss) [inline dev-go/go-md2man/go_md2man-1.0.7.recipe]:
```diff
-LICENSE="MIT"
+LICENSE="MIT
+	BSD (2-clause)"

File: dev-go/go-md2man/go_md2man-1.0.7.recipe [hunk 3]
```diff
@@ -0,0 +1,60 @@
+SUMMARY="Uses blackfriday to process markdown into man pages"
```
Comment (alaviss) [inline dev-go/go-md2man/go_md2man-1.0.7.recipe]: Here's a better `SUMMARY` for you

```diff
-SUMMARY="Uses blackfriday to process markdown into man pages"
+SUMMARY="A utility to convert markdown to man pages"

File: dev-go/go-md2man/go_md2man-1.0.7.recipe [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+DESCRIPTION="go-md2man uses blackfriday to process markdown into man pages"
```
Comment (alaviss) [inline dev-go/go-md2man/go_md2man-1.0.7.recipe]: And here's a better `DESCRIPTION` as well (from [go-md2man.1](https://github.com/cpuguy83/go-md2man/blob/master/go-md2man.1.md))

>go-md2man converts standard markdown formatted documents into manpages. It is written purely in Go so as to reduce dependencies on 3rd party libs.

**Comment by sfanxiang** [comment]
@alaviss 
> As they vendored blackfriday, this port is subjected to said project licenses and copyright

Thanks. I didn't notice this, and this recipe needs to be updated.

But I don't think putting two licenses there is a good idea, because people will think the project is dual-licensed, and that's probably not cpuguy83's intention.

**Comment by alaviss** [comment]
@sfanxiang 
> But I don't think putting two licenses there is a good idea, because people will think the project is dual-licensed, and that's probably not cpuguy83's intention.

It's not the matter of good or bad, it's for legal purposes. The [license][0] explictly requires the copyright notice, and the license text (which we currently don't do but instead refer to them with `LICENSE`, see https://github.com/haikuports/haikuporter/issues/156) to be included with the distribution.

[0]: https://github.com/cpuguy83/go-md2man/blob/8d868be6e9bf9d5350910bab97a050e49887600f/vendor/github.com/russross/blackfriday/LICENSE.txt#L13

**Comment by sfanxiang** [comment]
@alaviss 
You're right. I forgot about the static linkage here.

File: dev-go/go-md2man/go_md2man-1.0.7.recipe [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+	cmd:go_md2man = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
```
Comment (alaviss) [inline dev-go/go-md2man/go_md2man-1.0.7.recipe]: Please remove this blank line as required by our [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Comment (sfanxiang) [reply dev-go/go-md2man/go_md2man-1.0.7.recipe]: I can't find where it says "no blank line in BUILD_REQUIRES". Could you point it out?

Comment (alaviss) [reply dev-go/go-md2man/go_md2man-1.0.7.recipe]: From the [Guidelines][0]

> The correct ordering and spacing of the variables inside the recipe is as follows:

> ```
> BUILD_REQUIRES
> BUILD_PREREQUIRES
> ```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering

**Pull request merged**